src/sentry/static/sentry/app/stores/guideStore.tsx

```diff
@@ -0,0 +1,214 @@
+import {browserHistory} from 'react-router';
+import Reflux from 'reflux';
+
+import {Client} from 'app/api';
+import {Guide, GuidesServerData, GuidesContent} from 'app/components/assistant/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import ConfigStore from 'app/stores/configStore';
+import getGuidesContent from 'app/components/assistant/getGuidesContent';
+import GuideActions from 'app/actions/guideActions';
+import OrganizationsActions from 'app/actions/organizationsActions';
+
+const guidesContent: GuidesContent = getGuidesContent();
```

Comment:
Do you need to specify the type on this?

Reply:
What's your thought? I think it adds typing here: https://github.com/getsentry/sentry/pull/17569/files#diff-cc70d94737d46a81ec2ea09a62bae355R103